### PR TITLE
potential final CSS build, all unneeded code removed, comments added,…

### PR DESCRIPTION
**Pull request by DanielFMcKay**
… word tile background tweaked

### CODE REVIEW

File: assets/style.css [hunk 1]
```diff
@@ -114,6 +80,7 @@ header {
   margin: 0.25rem 0 1.25rem 0;
 }
 
+/* class for the dynamically generated color palette boxes */
```
Comment (KateMcRo) [inline assets/style.css:83]: This can be removed. The boxes are being generated at this size.

File: assets/style.css [hunk 2]
```diff
@@ -255,24 +223,11 @@ header {
...
   justify-content: center;
 }
 
-
-/* delete from final build */
-/* .colorTile p {
-  font-size: 1.25rem;
-} */
-
 .column1,
```
Comment (KateMcRo) [inline assets/style.css:231]: These could be given a shared class to set the same width for all.

File: assets/style.css (hunk 1 above)
Comment (DanielFMcKay) [reply assets/style.css:83]: okay, just changed it to be more accurate

File: assets/style.css [hunk 3]
```diff
@@ -309,12 +264,9 @@ header {
   background: transparent;
```
Comment (KateMcRo) [inline assets/style.css:264]: I believe this line can be removed.

**Review by KateMcRo — CHANGES_REQUESTED** [review]
Overall, looks good. Please see notes for minor changes that can be made.

File: assets/style.css (hunk 2 above)
Comment (DanielFMcKay) [reply assets/style.css:231]: I can do that if you're fine with me changing the HTML too

File: assets/style.css (hunk 3 above)
Comment (DanielFMcKay) [reply assets/style.css:264]: that's not actually in there x_X, that's just a note from github

**Pull request closed without merging**